src/player/ChunkLoader.ts

```diff
@@ -193,25 +194,30 @@ export class ChunkLoader {
     text: string,
     options: TTSModelOptions,
     context: AudioTextContext = {},
-  ): Promise<ArrayBuffer> {
+  ): Promise<AudioData> {
```

Comment:
@cursoragent no need to do the workarounds for the desiredFormat. Let's just return the data in the format its requested in and pass the buck on who needs to do the format handling